ui/app/components/app/input-number.js

```diff
@@ -52,32 +52,35 @@ InputNumber.prototype.setValue = function (newValue) {
   }
 }
 
-InputNumber.prototype.render = function () {
+InputNumber.prototype.render = function InputNumber () {
```

Comment:
Is this named `InputNumber` so that it'd be used as the `displayName`? I thought that the class being named `InputNumber` would accomplish that 🤔. I don't recall seeing the _render_ function being named the same as the component before.

Reply:
Yes. ESLint is confused<sup>[\[1\]][1]</sup> and doesn't recognize this class as a component and this is to appease ESLint. I'd rather this (temporarily) than a ESLint disable comment.

  [1]:https://github.com/yannickcr/eslint-plugin-react/blob/master/docs/rules/display-name.md#about-component-detection

Reply:
Cool! Good to know. I hadn't encountered that before.
At least it's just temporary.